.circleci/config.yml

```diff
@@ -133,7 +156,104 @@ jobs:
           root: ~/
           paths:
             - project
+            - dist/*.kar
+
+  smoke-test:
+    executor: smoke-test-executor
+
+    steps:
+      - attach_workspace:
+          at: ~/
   
+      - restore_cache:
+          keys:
+          # attempt to use the cache from the last smoke-test run
+          - v2-smoke-dependencies-{{ checksum "smoke-test/pom.xml" }}-{{ checksum "pom.xml" }}
+          # use the cache built for the same root pom
+          - v4-dependencies-{{ checksum "pom.xml" }}
+          # fallback to using the latest cache if no exact match is found
+          - v4-dependencies-
+          
+      - restore_cache:
+          keys:
+          - v1-docker-{{ checksum "smoke-test/src/main/resources/docker_fixed_images" }}
+  
+      # Link the cached maven repo
+      # Note: this is needed since we are using a circleci vm that is running as circleci whereas our images run as root
+      - run:
+          name: Link maven repo with cache
+          command: |
+            ln -s /tmp/.m2 ~/.m2
```

Comment:
see comment above, duplicated and different than above but does the same, a global command could help to ensure the same behavior.

Reply:
Will fix.